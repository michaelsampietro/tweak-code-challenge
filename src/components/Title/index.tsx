import React from 'react';

type TitleProps = {
  title: string;
}

const Title: React.FC<TitleProps> = ({ title }) => {
  return <h3>{title}</h3>
}

export default Title;