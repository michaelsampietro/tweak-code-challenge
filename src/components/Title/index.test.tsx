import { render } from "@testing-library/react";
import Title from ".";

describe('Title', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Title title='Test' />);
    expect(container).toMatchSnapshot();
  })
});