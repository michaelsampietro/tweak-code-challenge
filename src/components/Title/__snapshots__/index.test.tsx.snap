// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Should match snapshot 1`] = `
<div>
  <h3>
    Test
  </h3>
</div>
`;
